apps/prairielearn/elements/pl-rich-text-editor/pl-rich-text-editor.mustache

```diff
@@ -4,6 +4,12 @@ $(function() {
         readOnly: {{read_only}},
         placeholder: '{{placeholder}}',
         format: '{{format}}',
+        modules: {
+            counter: {
+                container: '#rte-counter-{{editor_uuid}}',
+                unit: '{{counter}}',
+            }
+        },
```

Comment:
Assuming this functionality moves out of a module: the container ID can be retrieved with the UUID that the code already has, so no need to add as an option. The unit could be an option directly (without hierarchy).